zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1423,21 +1423,99 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /variables:
+    post:
+      tags:
+        - Variable
+      operationId: createVariable
+      summary: Create variable
+      description: |
+        Create cluster level variables
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateVariableRequest"
+      responses:
+        "201":
+          description: >
+            The variable creation status.
+        "400":
+          $ref: "#/components/responses/InvalidData"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+
   /variables/{variableKey}:
-    get:
+    parameters:
+      - name: variableKey
+        in: path
+        required: true
+        description: The variable key.
+        schema:
+          type: string
+    patch:
       tags:
         - Variable
-      operationId: getVariable
-      summary: Get variable
+      operationId: updateVariable
+      summary: Update variables
       description: |
-        Get the variable by the variable key.
+        Update cluster level variables
       parameters:
         - name: variableKey
           in: path
           required: true
-          description: The variable key.
+          description: The key of the variable.
           schema:
             type: string
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/UpdateVariableRequest"
+      responses:
+        "204":
+          description: >
+            The variable update status.
+        "400":
+          $ref: "#/components/responses/InvalidData"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+    delete:
+      tags:
+        - Variable
+      operationId: deleteVariable
+      summary: Delete variable
+      description: |
+        Delete cluster level variables
+      responses:
+        "204":
+          description: >
+            The variable creation status.
```

🔧 We know the result, let's describe it like this.

```suggestion
            The variable was deleted successfully.
```